src/App.jsx

```diff
@@ -0,0 +1,23 @@
+import Home from "./components/pages/Home";
+import About from "./components/pages/About";
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+import Vans from "./components/pages/Vans";
+import VanDetail from "./components/pages/VanDetail";
+import "./server";
+
+function App() {
+  return (
+    <div>
```

Comment:
You can remove this div tag since there is nothing else other than the BrowserRouter in the component that is being returned.